tests/phpunit/CiviTest/CiviUnitTestCase.php

```diff
@@ -486,6 +486,17 @@ protected function assertPostConditions(): void {
     $this->validateAllContributions();
   }
 
+  /**
+   * Temporary pseudo-poly-fill for php8 to php9 transition.
```

Comment:
Should this read `phpunit8` and `phpunit9`?

Reply:
Yeah true - I'll fix it up if people want to procede with this approach